include/openssl/err.h.in

```diff
@@ -337,6 +337,7 @@ static ossl_unused ossl_inline int ERR_COMMON_ERROR(unsigned long errcode)
 # define ERR_R_OSSL_DECODER_LIB (ERR_LIB_OSSL_DECODER/* 60 */ | ERR_RFLAG_COMMON)
 
 /* Other common error codes, range 256..2^ERR_RFLAGS_OFFSET-1 */
+/* NOTE: MUST keep ERR_str_reasons in crypto/err/err.c in sync with this list */
```

Comment:
This is uninteresting for the public (this is a public header).  I see no reason for it to be here.

Reply:
Good point; removed this.